dev/com.ibm.ws.concurrent.mp_fat_jakarta/fat/src/com/ibm/ws/concurrent/mp/fat/jakarta/MPContextProp1_3_EE10_Test.java

```diff
@@ -34,8 +38,12 @@ public class MPContextProp1_3_EE10_Test extends FATServletClient {
     private static final String APP_NAME = "MPContextProp1_3_EE10_App";
 
     @Server("com.ibm.ws.concurrent.mp.fat.1.3.ee10")
+    @TestServlet(servlet = MPContextProp1_3_EE10_TestServlet.class, contextRoot = APP_NAME)
```

Comment:
Typically, when we write tests that avoid `@TestServlet` and put the `@Test` annotation on the test client side, it's because loading the servlet makes the test bucket fail when run on older Java levels due to something the servlet is compiled against that can't be loaded when the test is making the decision to skip.  Maybe that wasn't the reason here, but you should check what happens on older java levels to be sure.

Reply:
Interesting, I've not encountered that before. Will check into it.

Reply:
Tried to fix the FAT framework so that doesn't happen ... but it got messy so I pulled these changes out again.